.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+<!-- Be sure you are following the instructions in the
+[dev guidelines](http://www.numpy.org/devdocs/dev/gitwash/development_workflow.html) -->
+
+<!-- Include a properly formatted
+[commit message](http://www.numpy.org/devdocs/dev/gitwash/development_workflow.html#writing-the-commit-message). -->
```

Comment:
Note that this markdown will never render - but I suppose the point of markdown is it's readable even without formatting,

Reply:
removed markdown formatting